scripts/lc-builds/blueos_nvcc11_xl-2020.08.24.sh

```diff
@@ -7,7 +7,12 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ###############################################################################
 
-BUILD_SUFFIX=lc_blueos-nvcc10-xl_2019.06.12
+#
+## NOTE: After building code, you need to load the cuda 11 module to run
```

Comment:
Do you think it would be a good idea to echo this to the screen?

Reply:
We could do that. I didn't think about it. At this point, I just want to get this release out. I will merge this later today and do the release.